lib/iris/fileformats/pp.py

```diff
@@ -2426,8 +2443,12 @@ def save_pairs_from_cube(cube, field_coords=None, target=None):
         # From UM doc F3: "Set to -99 if LBEGIN not known"
         pp_field.lbuser[1] = -99
 
-        # Set the data
-        pp_field.data = slice2D.data
+        # Set the data, keeping it lazy where possible.
+        if slice2D.has_lazy_data():
+            slice_core_data = slice2D.lazy_data()
+        else:
+            slice_core_data = slice2D.data
```

Comment:
It's best like this because creating a simple wrapper around real data is substantially more costly in dask than it was with biggus :  The earliest dask branch retained existing cube property code to get cube.shape and cube.dtype from cube.lazy_data(), and then the tests took nearly twice as long!
Having said that, this code should be removed when we transition to dask anyway, as mentioned [above](https://github.com/SciTools/iris/pull/2452#discussion_r113162934)